static/js/controllers/inbox.js

```diff
@@ -32,6 +33,18 @@ var utils = require('kujua-utils'),
 
       require('../modules/manage-session').init();
 
+      $scope.isDistrict = function(row) {
```

Comment:
Not sure if these are in the right place. I put them on the scope so I can use with `filter:` in templates.